src/pages/home/HomePage.js

```diff
@@ -319,7 +312,7 @@ class App extends React.Component {
                                     style={[styles.appContent, styles.flex1, styles.flexColumn]}
                                 >
                                     <HeaderView
-                                        shouldShowNavigationMenuButton={this.state.isSmallScreenWidth}
+                                        shouldShowNavigationMenuButton={isSmallScreenWidth}
```

Comment:
NAB, but I think the HOC could be added to `HeaderView` and remove this prop entirely.

Reply:
Good suggestion, but I think I want to leave this for now just to limit the changes addressed in this PR.